src/ResourceManager/DataLakeStore/Commands.DataLakeStore/DataPlaneCommands/ExportAzureRmDataLakeStoreItem.cs

```diff
@@ -81,29 +76,25 @@ public class ExportAzureDataLakeStoreItem : DataLakeStoreFileSystemCmdletBase
             ParameterSetName = DiagnosticParameterSetName)]
         public SwitchParameter Resume { get; set; }
 
+        [Obsolete]
```

Comment:
@rahuldutta90 for all `Obsolete` attributes, please make sure you include a message that says what parameter you are deprecating in an upcoming breaking change release and what cmdlet it is in.

Reply:
Fixed. Also fixed this for ImportAzureRmDataLakeStoreItem.